### ISSUE TITLE
issue handling autocomplete k-on-close event

### ISSUE THREAD

**Issue by dpinart** [aurelia-ui-toolkits/aurelia-kendoui-bridge#723]
Hi,
I'm trying to handle autocomplete close event this way;
` <ak-autocomplete k-width="100%"
                           k-data-text-field="name"
                           k-data-source.bind="datasource"
                           k-filter="contains"
                           k-min-length.bind="2"
                           k-widget.two-way="autoComplete"
                           k-value.bind="cityName"
                           k-on-close.delegate="onCloseCitySelector($event.detail)">
          </ak-autocomplete>`

even if my onCloseCitySelector does nothing, only by handling the onClose event prevents the popup to close. I have to call e.sender.close() on onCloseCitySelector method in order to force the popup to be closed but I'm seeing a "RangeError: Maximum call stack size exceeded"

Is there any way to prevent this error?

**Comment by JeroenVinke** [comment]
Could you create a gist run that shows this issue? Instructions can be found [here](http://aurelia-ui-toolkits.github.io/demo-kendo/#/help). Thanks

**Comment by pKrav75** [comment]
Hello i may be wrong, but aurelia default behavior when processing event is to call preventDefault, in this case it will instruct Kendo not to close the container. See https://github.com/aurelia/binding/issues/336 and the solution to control this behavior is to return true in the handler.
Again not 100% sure of that

**Comment by JeroenVinke** [comment]
Correct! That was the cause and this was fixed in the latest version of the bridge
